puppet/modules/slave/manifests/postgresql.pp

```diff
@@ -2,59 +2,24 @@
 class slave::postgresql {
   # only CentOS slaves are used to run unit tests
   if $facts['os']['family'] == 'RedHat' {
-    if $facts['os']['release']['major'] == '7' {
-      ['postgresql-server', 'postgresql-devel', 'postgresql-client', 'postgresql'].each |$pkg| {
-        package { "${pkg}-nonscl":
-          ensure => absent,
-          name   => $pkg,
-          before => Class['postgresql::globals'],
-        }
-      }
-
-      if $facts['os']['name'] == 'CentOS' {
-        package { 'centos-release-scl-rh':
-          ensure => 'present',
-          before => Class['postgresql::globals'],
-        }
-      } elsif $facts['ec2_metadata'] {
-        yumrepo { 'rhel-server-rhui-rhscl-7-rpms':
-          enabled => true,
-          before  => Class['postgresql::globals'],
-        }
-      }
-
-      yumrepo { 'foreman-infra-el7':
-        descr    => 'foreman-infra-el7',
-        baseurl  => 'https://yum.theforeman.org/infra/el7/',
-        enabled  => true,
-        gpgcheck => false,
-      } ->
-      package { ['rh-postgresql12-postgresql-debversion', 'rh-postgresql12-postgresql-evr']:
-        ensure  => 'present',
-        notify  => Class['postgresql::server::service'],
-        require => Class['postgresql::server::install'],
-      }
-
-      class { 'postgresql::globals':
-        version              => '12',
-        client_package_name  => 'rh-postgresql12-postgresql-syspaths',
-        devel_package_name   => 'rh-postgresql12-postgresql-devel',
-        server_package_name  => 'rh-postgresql12-postgresql-server-syspaths',
-        contrib_package_name => 'rh-postgresql12-postgresql-contrib-syspaths',
-        service_name         => 'postgresql',
-        datadir              => '/var/opt/rh/rh-postgresql12/lib/pgsql/data',
-        confdir              => '/var/opt/rh/rh-postgresql12/lib/pgsql/data',
-        bindir               => '/usr/bin',
-      }
-
-      file { '/etc/profile.d/enable_postgresql12_scl.sh':
-        ensure  => absent,
+    ['postgresql-server', 'postgresql-devel', 'postgresql-client', 'postgresql'].each |$pkg| {
+      package { "${pkg}":
+        ensure => absent,
+        name   => $pkg,
+        before => Class['postgresql::globals'],
       }
+    }
 
-      file { '/usr/bin/pg_config':
-        ensure => 'link',
-        target => '/opt/rh/rh-postgresql12/root/usr/bin/pg_config',
-      }
+    yumrepo { 'foreman-infra-el8':
+      descr    => 'foreman-infra-el8',
+      baseurl  => 'https://yum.theforeman.org/infra/el8/',
```

Comment:
I am going to have to create this.

Reply:
Actually, I think creating a Copr repository for this purpose makes more sense and is easier?

Reply:
Why not just using the katello repos?

Reply:
I had the same thought and then... why didn't we do that before? I assume we had a "good" reason

Reply:
Because they're in the Pulp repos. This is what we do elsehwere: https://github.com/theforeman/puppet-katello/blob/5585d1114bcd9ad5e04f3bde074d7fcef6498696/spec/setup_acceptance_node.pp#L19-L26